itemadapter/utils.py

```diff
@@ -1,6 +1,45 @@
+from types import MappingProxyType
 from typing import Any
 
 
+__all__ = [
+    "get_class_field_meta",
+    "is_attrs_instance",
+    "is_dataclass_instance",
+    "is_item",
+    "is_scrapy_item",
+]
```

Comment:
+0

If someone uses an asterisk import and uses _-prefixed functions imported that way, they deserve whatever happens to them :slightly_smiling_face:

Reply:
hahaha fair enough